main.go

```diff
@@ -48,6 +50,14 @@ func main() {
 	}
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&zapOpts)))
 
+	privilegedClient, err := repositories.BuildClient(k8sClientConfig)
+	if err != nil {
+		panic(fmt.Sprintf("could not create privileged k8s client: %v", err))
+	}
+
+	orgRepo := repositories.NewOrgRepo(config.RootNamespace, privilegedClient)
+	orgHandler := apis.NewOrgHandler(orgRepo, config.ServerURL)
```

Comment:
You could inline this into the handlers slice below like the other handlers for consistency.

Reply:
Agree